app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command):
+    sourse = command.split()[1]
+    target = command.split()[2]
+
+    if "/" in target:
+        target_split = target.split("/")
+        target_path = os.path.join(*target_split)
```

Comment:
you split the `target` and then join it, so you get exactly what you had without any difference